docs/python-examples/fixtures/test_fixtures.py

```diff
@@ -0,0 +1,109 @@
+import logging
+
+import pytest
+
+logger = logging.getLogger(__name__)
+
+
+def alter_pvc(pvc):
+    """
+    Dummy helper function to alter pvc
+
+    Args:
+        pvc (PVC): PVC object reference
+    """
+    logger.info(f"Doing something with pvc: {pvc}")
+
+
+@pytest.fixture(scope='class')
+def three_pvcs(pvcs_factory, storage_class):
+    """
+    This is fixture related just to one specific test in this module so no
+    need to have it defined in conftest
+    """
+    return pvcs_factory(3, storage_class)
```

Comment:
`pvcs_factory` only returns one pvc at a time, you'll need to call it several times in this example or update the fixture definition to support passing a count. I know these are only examples but I can see someone getting confused by this. Same goes for the example test cases below that use the factory.

Reply:
Pvcs_factory returns the function which is called here and parametrized to give you three pvcs.

Reply:
I see that now, I was looking at the wrong definition, scratch this!